FeatureToggle.Application/Requests/Commands/UserCommands/AddUserCommandValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentValidation;
+
+namespace FeatureToggle.Application.Requests.Commands.UserCommands
+{
+    public class AddUserCommandValidator : AbstractValidator<AddUserCommand>
```

Comment:
put command, commandHandler, validator in a single folder

Reply:
noted